tensorflow-examples/src/main/java/org/tensorflow/model/examples/cnn/fastrcnn/FasterRcnnInception.java

```diff
@@ -269,7 +265,9 @@ public static void main(String[] params) {
                 //The given SavedModel SignatureDef input
                 feedDict.put("input_tensor", reshapeTensor);
                 //The given SavedModel MetaGraphDef key
-                Map<String, Tensor> outputTensorMap = model.function("serving_default").call(feedDict);
+                Map<String, Tensor> outputTensorMap = new HashMap<>();
+                model.function("serving_default").call(feedDict);
```

Comment:
This change means the output of the model is discarded, please revert it.

Reply:
I roughly understand what you mean, but I'm not sure if this is correct. `get(0)` will return a tensor.